<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogaddticket</class>
 <widget class="QDialog" name="dialogaddticket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TicketAdd</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="1">
      <widget class="QtMaterialAutoComplete" name="BalineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ealabel">
       <property name="text">
        <string>终点站</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QtMaterialAutoComplete" name="EtlineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="etlable">
       <property name="text">
        <string>到站时间</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="ticketlabel">
       <property name="text">
        <string>车票数</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QtMaterialAutoComplete" name="BtlineEdit"/>
     </item>
     <item row="6" column="1">
      <widget class="QtMaterialAutoComplete" name="TicketlineEdit"/>
     </item>
     <item row="9" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="7" column="1">
      <widget class="QtMaterialAutoComplete" name="PrlineEdit"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>票价</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QtMaterialRaisedButton" name="addbtn">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(87, 253, 255);</string>
         </property>
         <property name="text">
          <string>save</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QtMaterialRaisedButton" name="canclebtn">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(87, 253, 255);</string>
         </property>
         <property name="text">
          <string>cancle</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="btlabel">
       <property name="text">
        <string>出发时间</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="balabel">
       <property name="text">
        <string>始发站</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QtMaterialAutoComplete" name="IdlineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="idlabel">
       <property name="text">
        <string>列车号</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QtMaterialAutoComplete" name="EalineEdit"/>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtMaterialAutoComplete</class>
   <extends>QLineEdit</extends>
   <header location="global">qtmaterialautocomplete.h</header>
  </customwidget>
  <customwidget>
   <class>QtMaterialRaisedButton</class>
   <extends>QPushButton</extends>
   <header location="global">qtmaterialraisedbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
